src/commonTest/kotlin/UuidTest.kt

```diff
@@ -44,9 +44,8 @@ class UuidTest {
         assertEquals(-4287190811922382213, uuid.mostSignificantBits)
     }
 
-    @Test
-    fun generates_a_UUID_with_correct_version_and_variant_bits() {
-        val uuid = Uuid()
+    @Test fun uuid4() {
```

Comment:
This should be renamed to something like generates_a_uuid4 to avoid naming conflict with the new uuid4 function

Reply:
A naming conflict is not possible, this function is part of the `UuidTest` class.

Reply:
😝 of course, yeah.